examples/platform_channel/macos/Runner/BatteryLevelPlugin.swift

```diff
@@ -0,0 +1,76 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import FlutterMacOS
+import Foundation
+
+class BatteryLevelPlugin: NSObject, FlutterPlugin, FlutterStreamHandler,
```

Comment:
Good point - I'll integrate the stuff in BatteryPlugin into the main code. I think it still makes sense to keep the battery-specific logic in its own class since on iOS it's already isolated in a built-in class and overall is irrelevant to the general mechanics of using the channels.

Reply:
Done.